docs/benchmarks/django-workload.md

```diff
@@ -0,0 +1,352 @@
+-XX:+CMSClassUnloadingEnabled
+
+# Additional settings
+-XX:+UseCondCardMark
+-XX:MaxTenuringThreshold=2
+-XX:-UseBiasedLocking
+-XX:+UseTLAB
+-XX:+ResizeTLAB
+-XX:+PerfDisableSharedMem
+-XX:+AlwaysPreTouch
+-XX:+UnlockDiagnosticVMOptions
+-XX:ParGCCardsPerStrideChunk=4096
+```
+
+## Memcached configuration
+The cloudsuite/memcached-webtier docker sets up a memcached server with 5GB memory; you'll need a server or VM with that amount of memory.
+The server binds to all network interfaces so this should only be run in a firewalled environment.
+
+### Memcached Docker status/validation
+To check the status of memcached container, execute the below script on the memcached host machine
+```
+cloudsuite/benchmarks/django-workload/check_containers.sh <memcached-host-private-ip> 11211 memcached
+```
+
+## Graphite configuration
+
+### Graphite Docker status/validation
+To check the status of Graphite container, execute the below script on the Graphite host machine
+```
+cloudsuite/benchmarks/django-workload/check_containers.sh <graphite-host-private-ip> 80 graphite
+```
+
+### Mandatory config
+The default config of the hopsoft/graphite-statsd docker container will likely
+cause your storage space to run out because of the amount of data being logged
+into statsd by the Django Workload. In order to solve this, please perform the
+steps below after starting the container. All commands should be run as root.
+
+Obtain a shell in the container:
+```
+sudo docker exec -it graphite bash
+cd opt/graphite/conf/
+```
+
+Add the following line to `blacklist.conf`:
+```
+^stats[^.]*\.benchmarkoutput\.
+```
+
+Edit the `carbon.conf` file to enable whitelisting. Search for the line
+containing `USE_WHITELIST`, uncomment it and set it to True:
+```
+USE_WHITELIST = True
+```
+
+Edit the retention policy in `storage-schemas.conf`:
+```
+[default_1min_for_1day]
+pattern = .*
+retentions = 10s:2h,1min:2d,10min:14d
+```
+
+Exit the docker container and restart it for the configurations to take effect:
+```
+exit
+docker stop graphite
+docker start graphite
+```
+
+Should the disk space fill up again, you can simply delete graphite’s database:
+```
+docker exec -it graphite bash
+rm –rf /opt/graphite/storage/whisper/*
+```
+
+## uWSGI configuration
+
+### uWSGI Docker status/validation
+To check the status of uWSGI container, execute the below script on the uWSGI host machine
+```
+cloudsuite/benchmarks/django-workload/check_containers.sh <uWSGI-host-private-ip> 8000 uwsgi
+```
+
+### Build uWSGI Image with custom Python build
+
+        $ ./build_uwsgi.sh [/absolute/path/to/cpython/install]
+
+### Debug
+
+#### uWSGI Logging
+Logging for uWSGI is turned off by default for benchmarking purposes. In order
+to turn it back on, comment out the `disable-logging` parameter in uwsgi.ini,
+like this:
+```
+#disable-logging = True
+```
+
+#### Django debugging
+If you get HTTP response codes different than 200, change the DEBUG parameter
+in cluster_settings.py:
```

Comment:
It would be good to clearly introduce how to access 'cluster_settings.py'.

Reply:
Will update the readme